packages/framework/src/Form/Admin/Product/ProductFormType.php

```diff
@@ -57,7 +57,6 @@ class ProductFormType extends AbstractType
     const VALIDATION_GROUP_NOT_USING_STOCK = 'notUsingStock';
 
     const CSRF_TOKEN_ID = 'product_edit_type';
-    const VALIDATION_GROUP_MANUAL_PRICE_CALCULATION = 'manualPriceCalculation';
```

Comment:
Půjde odstranit lehce i `VALIDATION_GROUP_AUTO_PRICE_CALCULATION`?

Reply:
Yes, it is possible to remove the group as well - I will add a fixup commit :+1: